src/common/components/vanilla/button/button.css

```diff
@@ -0,0 +1,3 @@
+.button {
+  font-family: "Ubuntu", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
Vanilla defined separate font family for buttons (or other components)?

This seems like a good candidate for moving to variables.